decidim-meetings/app/views/decidim/meetings/admin/meetings/index.html.erb

```diff
@@ -77,7 +77,22 @@
                 <%= resource_permissions_link(meeting) %>
 
                 <% if allowed_to? :destroy, :meeting, meeting: meeting %>
-                  <%= icon_link_to "circle-x", meeting_path(meeting), t("actions.destroy", scope: "decidim.meetings"), method: :delete, class: "action-icon--remove", data: { confirm: t("actions.confirm_destroy", scope: "decidim.meetings") } %>
+                  <% if present(meeting).proposals.empty? %>
+                    <%= icon_link_to "circle-x", meeting_path(meeting), t("actions.destroy", scope: "decidim.meetings"), method: :delete, class: "action-icon--remove", data: { confirm: t("actions.confirm_destroy", scope: "decidim.meetings") } %>
+                  <% else %>
+                    <%=
+                      content_tag(:button,
+                                  class: ["action-icon", "action-icon--remove", "destroy-meeting-alert"],
+                                  "data-invalid-destroy-message" => t("actions.invalid_destroy.proposals_count.#{present(meeting).proposals.count == 1 ? "one" : "other"}", scope: "decidim.meetings", proposals_count: present(meeting).proposals.count),
```

Comment:
avoid many queries
```suggestion
                                  "data-invalid-destroy-message" => t("actions.invalid_destroy.proposals_count.#{present(meeting).proposals.size == 1 ? "one" : "other"}", scope: "decidim.meetings", proposals_count: present(meeting).proposals.size),
```

Reply:
Also we can use the standard I18n pluralization API: https://guides.rubyonrails.org/i18n.html#pluralization